### ISSUE TITLE
img alt text test disabled in html-validation test suite

### ISSUE THREAD

**Issue by andymantell** [LandRegistry/land-registry-elements#46]
Due to changes upstream in htmllint (See https://github.com/htmllint/htmllint/issues/148) I have had to disable the alt text check as it is causing failures with empty alt texts (Such as on the govuk crown in this case). Need to monitor the situation upstream in htmllint and re-enable this test when we can.
